packages/client-core/src/admin/components/Setting/Server.tsx

```diff
@@ -0,0 +1,222 @@
+            <InputBase name="mode" className={classes.input} style={{ color: '#fff' }} />
+          </Paper>
+          <label> Host Name</label>
+          <Paper component="div" className={classes.createInput}>
+            <InputBase name="hostname" className={classes.input} style={{ color: '#fff' }} />
+          </Paper>
+          <label>Port</label>
+          <Paper component="div" className={classes.createInput}>
+            <InputBase name="port" className={classes.input} style={{ color: '#fff' }} />
+          </Paper>
+          <label> Client Host</label>
+          <Paper component="div" className={classes.createInput}>
+            <InputBase name="clienthost" className={classes.input} style={{ color: '#fff' }} />
+          </Paper>
+          <label>Root Directory</label>
+          <Paper component="div" className={classes.createInput}>
+            <InputBase name="rootDir" className={classes.input} style={{ color: '#fff' }} />
+          </Paper>
+          <label>Public Directory</label>
+          <Paper component="div" className={classes.createInput}>
+            <InputBase name="publicDir" className={classes.input} style={{ color: '#fff' }} />
+          </Paper>
+          <label>Node Modules Directory</label>
+          <Paper component="div" className={classes.createInput}>
+            <InputBase name="nodeModule" className={classes.input} style={{ color: '#fff' }} />
+          </Paper>{' '}
+          <label>Local StorageProvider </label>
+          <Paper component="div" className={classes.createInput}>
+            <InputBase name="localStorageProvider" className={classes.input} style={{ color: '#fff' }} />
+          </Paper>
+          <label> Perform Dry Run</label>
+          <Paper component="div" className={classes.createInput}>
+            <Switch
+              checked={dryRun.checkedB}
+              onChange={handleDryRun}
+              color="primary"
+              name="checkedB"
+              inputProps={{ 'aria-label': 'primary checkbox' }}
+            />
+          </Paper>
+        </Grid>
+        <Grid item xs={12} sm={6}>
+          <label>Storage Provider </label>
+          <Paper component="div" className={classes.createInput}>
+            <InputBase name="StorageProvider" className={classes.input} style={{ color: '#fff' }} />
+          </Paper>
+          <label>Get Tracking Id </label>
+          <Paper component="div" className={classes.createInput}>
+            <IconButton>
+              <Icon icon="emojione:key" />
+            </IconButton>
+            <InputBase
+              name="StorageProvider"
+              className={classes.input}
+              style={{ color: '#fff' }}
+              defaultValue="wwtrw766666666676aasddsadasdwww"
+            />
+            <IconButton>
+              <Icon icon="heroicons-outline:clipboard-copy" color="orange" />
+            </IconButton>
+            <IconButton onClick={handleLoad}>
```

Comment:
Don't need a refresh button - this is a Google Analytics tracking ID that will be entered, not an ID that the platform is (re-)generating.